src/js/helpers/pagination-helper.js

```diff
@@ -0,0 +1,5 @@
+export function getFirstPageLoader (filter, requestFunction) {
```

Comment:
Do you use this method anywhere? If no, you don't need it.

Reply:
Fixed